contracts/dao/voting/lockdrop-vault/src/tests.rs

```diff
@@ -65,6 +76,116 @@ fn mock_app() -> App {
     })
 }
 
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone)]
+#[serde(rename_all = "snake_case")]
+pub struct EmptyMsg {}
+
+fn lockdrop_query(_deps: Deps, _env: Env, msg: LockdropQueryMsg) -> StdResult<Binary> {
+    match msg {
+        LockdropQueryMsg::QueryUserLockupTotalAtHeight {
+            pool_type,
+            user_address: _,
+            height: _,
+        } => {
+            let response = match pool_type {
+                PoolType::ATOM => Uint128::from(1000u64),
+                PoolType::USDC => Uint128::from(2000u64),
+            };
+
+            to_binary(&response)
+        }
+        LockdropQueryMsg::QueryLockupTotalAtHeight {
+            pool_type,
+            height: _,
+        } => {
+            let response = match pool_type {
+                PoolType::ATOM => Uint128::from(3000u64),
+                PoolType::USDC => Uint128::from(4000u64),
+            };
+
+            to_binary(&response)
+        }
+        _ => panic!("DO NOT ENTER HERE"),
```

Comment:
There is a better macro for that:
```suggestion
        _ => unreachable!(),
```